src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.config;
+
+import org.folio.converter.RecordToQuickMarcConverter;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackageClasses = RecordToQuickMarcConverter.class)
```

Comment:
One component out of spring context. Probably it should be
`@ComponentScan(basePackageClasses = { RecordToQuickMarcConverter.class, QuickMarcToRecordConverter.class })
`

Reply:
@grabsefx, it's OK now. We are going to use basePackages in the nearest future :)

Reply:
`@ComponentsScan`'s `basePackageClasses` parameter requires just one class to scan the whole package, it's a better solution than `basePackages` because it is free from typos and packages renaming :)

Reply:
Sure. But we will have not only one component. Let's see how it goes. Let's skip it for now.

Reply:
In this case, please, consider https://github.com/folio-org/mod-quick-marc/pull/4#discussion_r397609680